JCoreRN.podspec

```
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.source             = { :git => "https://github.com/jpush/jcore-react-native.git", :tag => "#{s.version}" }
```

RuboCop (1.81.1)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)